src/index.ts

```diff
@@ -139,6 +139,27 @@ export const stockItemRulesFormWorkspace = getAsyncLifecycle(
   () => import('./stock-items/add-stock-item/stock-item-rules/add-stock-rules.component'),
   options,
 );
+
+export const stockSourcesFormWorkspace = getAsyncLifecycle(
+  () => import('./stock-sources/add-stock-sources/add-stock-sources.workspace'),
+  options,
+);
+
+export const stockLocationsFormWorkspace = getAsyncLifecycle(
+  () => import('./stock-locations/add-locations-form.component'),
```

Comment:
```suggestion
  () => import('./stock-locations/add-locations-form.workspace'),
```

This component should use the `workspace` suffix.